src/test/regress/expected/multi_test_helpers_superuser.out

```diff
@@ -38,6 +38,7 @@ SELECT dist_node_check.matches AND dist_placement_check.matches
 FROM dist_node_check CROSS JOIN dist_placement_check
 $$;
 -- partition_task_list_results tests the internal PartitionTasklistResults function
+SET citus.enable_metadata_sync TO OFF;
```

Comment:
any reason for this change?

Reply:
We check non-existency of that function within tests.